src/styles/maincards.scss

```diff
@@ -0,0 +1,125 @@
+.main {
+  display: flex;
+  max-width: 1044px;
+  min-width: 1000px;
+  margin: auto;
+}
+
+.card-container {
+  padding-top: 50px;
+  display: flex;
+  flex-wrap: wrap;
```

Comment:
Please check if a new card is added, it should be displayed below the leftmost one, in order
![image](https://user-images.githubusercontent.com/54936044/224473200-263bf8d8-6d13-4d3e-a906-1e5bb0d4b1b9.png)